src/client/app/utils.ts

```diff
@@ -11,6 +11,7 @@ export function joinPath(base: string, path: string): string {
 }
 
 export function withBase(path: string) {
+  if (/^(https?:)?\/\//.test(path)) return path
```

Comment:
let's move the regex to a const outside `const EXTERNAL_URL_RE = /^https?:/i`. Which should be enough (using `//external.com` is an antipattern and should be avoided

Then use a ternary: `return re.test(path) ? path : join(...)

Reply:
Great